src/Nri/Ui/Highlighter/V1.elm

```diff
@@ -134,7 +134,7 @@ init config =
     , hasChanged = NotChanged
     , selectionStartIndex = Nothing
     , selectionEndIndex = Nothing
-    , focusIndex = 0
+    , focusIndex = 2
```

Comment:
I think it's _incidentally_ true that the first focusable element in the example text is the second position, but I don't think it's guaranteed, unfortunately.